docs/data/joy/main-features/color-inversion/ColorInversionNavigation.js

```diff
@@ -161,28 +166,27 @@ export default function ColorInversionNavigation() {
             vertical: 'bottom',
             horizontal: 'right',
           }}
-          badgeInset="14%"
-          sx={{ [`& .${badgeClasses.badge}`]: { bgcolor: 'success.300' } }}
+          badgeInset="20%"
```

Comment:
```suggestion
          badgeInset="15%"
```

This felt a bit better than 20, to me — a little less overlapping with the avatar.